mailing_list/models.py

```diff
@@ -70,7 +77,8 @@ def get_or_create_mailing_lists_for_canvas_course_id(self, canvas_course_id, **k
                 if not listserv_list:
                     listserv_client.create_list(mailing_list)
                 else:
-                    access_level = listserv_list['access_level']
+                    # fetch  access level that is now  stored in the DB
+                    access_level = mailing_list.access_level
```

Comment:
You could remove this else block now and just do `'access_level': mailing_list.access_level` on line 91 below.